.conda/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set pyproject = load_file_data('../pyproject.toml', from_recipe_dir=True) %}
+{% set project = pyproject.get('project') %}
+{% set urls = pyproject.get('project', {}).get('urls') %}
+{% set version = environ.get('BUILD_VERSION', '0.8.0a0') %}
+package:
+  name: {{ project.get('name') }}
+  version: {{ version }}
+
+source:
+  fn: {{ project.get('name') }}-{{ version }}.tar.gz
+  url: ../dist/{{ project.get('name') }}-{{ version }}.tar.gz
+
+build:
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python>=3.8, <4.0
+    - setuptools
+
+  run:
```

Comment:
`{{ project.get('dependencies') }}` does not work because of `pypdfium2_helpers` instead of `pypdfium2` ?

Reply:
assumably correct I didn't even try since there is no link from pypi to the conda channel & package

Reply:
Hmm, indeed the name incongruence turns out to be unfortunate here.

Is there any chance of replacing `pypdfium2` by `pypdfium2_helpers` after the `project.get(...)` via jinja dynamics?

Reply:
Perhaps just like below with the summary ?
`{{ project.get('dependencies') | replace("pypdfium2", "pypdfium2_helpers") }}`
But I don't know if that works with lists...

Reply:
@frgfm  ? :)